qa/tasks/cephfs/test_misc.py

```diff
@@ -290,6 +291,71 @@ def test_file_sync_stuck_for_around_5s(self):
         dir_path = "file_sync_do_not_wait_mdlog_testdir"
         self._test_sync_stuck_for_around_5s(dir_path, True)
 
+    def test_file_filesystem_sync_crash(self):
+        """
+        To check whether will the kernel crash when doing the file/filesystem sync.
+        """
+
+        stop_thread = False
+        dir_path = "file_filesystem_sync_crash_testdir"
+        self.mount_a.run_shell(["mkdir", dir_path])
+
+        def mkdir_rmdir_thread(mount, path):
+            #global stop_thread
+
+            log.info(" mkdir_rmdir_thread starting...")
+            num = 0
+            while not stop_thread:
+                n = num
+                m = num
+                for i in range(10):
+                    mount.run_shell(["mkdir", os.path.join(path, f"{n}")])
+                    n += 1
+                for i in range(10):
+                    mount.run_shell(["rm", "-rf", os.path.join(path, f"{m}")])
+                    m += 1
+                num += 10
+            log.info(" mkdir_rmdir_thread stopped")
+
+        def filesystem_sync_thread(mount, path):
+            #global stop_thread
+
+            log.info(" filesystem_sync_thread starting...")
```

Comment:
nit: space at the beginning intended ? Same comment for below instances.

Reply:
Yeah, that was intended.